src/main/resources/bidder-config/kueezrtb.yaml

```diff
@@ -0,0 +1,21 @@
+adapters:
+  kueezrtb:
+    endpoint: https://prebidsrvr.kueezrtb.com/openrtb/
+    endpoint-compression: gzip
+    ortb-version: "2.6"
```

Comment:
As I see from GO, it's an 2.5 bidder, but it supports gpp string, so just remove this line and it will be fine